src/analyze/analyze.c

```diff
@@ -2277,6 +2281,10 @@ static int parse_argv(int argc, char *argv[]) {
                 case 'h':
                         return help(0, NULL, NULL);
 
+                case ARG_ERROR_ON:
+                        arg_error_on = optarg;
```

Comment:
The mode should not be passed around as a string internally. Please add a string table and convert it to an enum value here.